.gitignore

```diff
@@ -26,3 +26,8 @@ telepresence.log
 
 # distutils gunk
 leastauthority.com.egg-info/
+
+# env file
+.env
+
+.DS_Store
```

Comment:
Thanks for adding this ignore.  Can you also delete the `.DS_Store` files that were added?